web/src/toga_web/widgets/scrollcontainer.py

```diff
@@ -0,0 +1,78 @@
+from .base import Widget
+
+
+class ScrollContainer(Widget):
+    def __init__(self, interface):
+        super().__init__(interface)
+        self.native.addEventListener("scroll", self._on_native_scroll)
```

Comment:
Convention is to use `dom_` as a prefix, with the method name matching the native event name; so this should be `dom_onscroll`.